02/app.js

```diff
@@ -1,25 +1,31 @@
-import React from 'react';
-import { createRoot } from 'react-dom/client';
+import React from "react";
+import { createRoot } from "react-dom/client";
 
-import List from './List';
-import Form from './Form';
+import List from "./List";
+import Form from "./Form";
 
 class App extends React.Component {
-    state = {
-        usersList: [],
-    }
+  state = {
+    usersList: [],
+  };
 
-    render() {
-        const  { usersList } = this.state;
+  addUser = (name) => {
+    this.setState({ usersList: this.state.usersList.concat(name) }, () => {
```

Comment:
Jeśli chcemy korzystać z `id` to powinniśmy to robić na tym etapie tj. `this.state.usersList.concat({name: name, `id`: crypto.randomUUID();})` - w ten sposób każdy użytkownik ma swoje id i nie będzie ono zmieniane po każdym `render()`